### PR TITLE
Set slf4j dependencies version in root pom.

### PR DESCRIPTION
**Pull request by groldan**
Homogenize slf4j dependency version, there were
several mixed versions across modules.

Upgrade to latest stable 1.7.14.

### CODE REVIEW

File: src/pom.xml [hunk 1]
```diff
@@ -844,8 +844,18 @@
    </dependency>
    <dependency>
      <groupId>org.slf4j</groupId>
+     <artifactId>slf4j-api</artifactId>
+     <version>1.7.14</version>
```
Comment (jodygarnett) [inline src/pom.xml]: Thanks for adding to dependency management section.

(I agree with only two entries it is not worth breaking out a ``${slfj.version}`` property.

**Comment by jodygarnett** [comment]
Thanks for the build improvement.

**Comment by jodygarnett** [comment]
@groldan - noticed this was to the 2.7.x branch, can I ask you to cherry pick this change to 2.8.x and master.

**Comment by groldan** [comment]
@jodygarnett sure, thanks, my next question was going to be if I should create separate PR's for 2.8.x and master.

**Comment by jodygarnett** [comment]
Just do it.
On Mon, Feb 1, 2016 at 5:56 PM Gabriel Roldan <notifications@github.com>
wrote:

> @jodygarnett <https://github.com/jodygarnett> sure, thanks, my next
> question was going to be if I should create separate PR's for 2.8.x and
> master.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoserver/geoserver/pull/1461#issuecomment-178066317>.
>
-- 
--
Jody Garnett

**Pull request merged**